webapp/components/Editor/Editor.vue

```diff
@@ -212,8 +213,9 @@ export default {
     },
     sanitizeQuery(query) {
       if (this.suggestionType === HASHTAG) {
-        // remove all not allowed chars
-        query = query.replace(/[^a-zA-Z0-9]/gm, '')
+        // remove all unallowed chars
+        const regX = build('[^\\pL0-9]')
```

Comment:
same as above – I could go check what this regex does exactly but would prefer to see it directly in the code, either as variable name or, since there's a comment already, as explanation in the comment

Reply:
Wonderful 😍

@mattwr18 did already … 😍
… and I changed the name again for clarification.